requirements.txt

```diff
@@ -7,5 +7,5 @@ gunicorn==20.1.0
 requests==2.26.0
 retry==0.9.2
 sentry-sdk==1.1.0
-hm-pyhelper==0.11.13
 dbus-python==1.2.16
+hm_pyhelper==0.12.0
```

Comment:
Think this should be hm-pyhelper and there's a v0.12.1 now too FYI

Reply:
Oh, this is interesting, it worked with hm_pyhelper as it didn't fail on me. I'll change it to hm-pyhelper though as that's the correct name for the package.

Reply:
@SebastianMaj actually @kashifpk said it would work fine with underscore instead as it uses them somewhat interchangeably but just thought the "proper" name is better and it's what we use in all other repos AFAIK so just for consistency